### DISCUSSION TITLE
How to detect multiple faces?

### DISCUSSION THREAD

**Discussion by mctrafik** [vladmandic/human#296] [Q&A] [answered]
Hello.

No matter what I try I only ever get 1 face in the result. 

Config:

```typescript
const config: Partial<Config> = {
  modelBasePath: `file://${resolve("./node_modules/@vladmandic/human/models")}`,
  debug: true,
  face: {
    enabled: true,
    // modelPath: "blazeface.json", // faceres.json
    description: { enabled: true },
    mesh: { enabled: true }, // Required to find anything for some reason.
    liveness: { enabled: false },
    antispoof: { enabled: false },
    attention: { enabled: false },
    gear: { enabled: false },
    detector: { enabled: true, rotation: false },
    iris: { enabled: false },
    emotion: { enabled: false },
  },
  hand: { enabled: false },
  body: { enabled: false },
  object: { enabled: false },
  gesture: { enabled: false },
};
````

And faces is always an array with 1 face.
I tried calling `human.next(result)` but still, only the one face.
please help.

Image for reference:
![foo](https://user-images.githubusercontent.com/1694347/190084637-2d0f7671-16e6-421b-8f59-99daf89d7b78.jpg)

**Comment by mctrafik** [comment, marked as answer]
Eh... looks like this fixes its: `{ ..., detector: { maxDetected: 100 }, ... }`  which I thought was the default. It's really odd that it's defaulting to `1` . I saw in this wiki article: https://github.com/vladmandic/human/wiki/Embedding that it will just detect multiple faces without specifying `maxDetected` in the config.

**Comment by vladmandic** [comment]
@mctrafik You are correct on the required config. Btw, I've changed the default value for `maxDetected` to 1 as the most common use case for `Human` was with selfie camera or with webcam with single person in the frame. I'm going to update the docs to reflect this.
